src/applications/appeals/onramp/constants/display-conditions/section-2.js

```diff
@@ -54,17 +66,19 @@ export const section2DCs = Object.freeze({
   Q_2_H_2A_JUDGE_HEARING: {
     Q_1_1_CLAIM_DECISION: YES,
     Q_1_2_CLAIM_DECISION: YES,
-    Q_1_3_CLAIM_CONTESTED: NO,
-    Q_2_0_CLAIM_TYPE: HLR,
-    Q_2_H_1_EXISTING_BOARD_APPEAL: NO,
     Q_2_H_2_NEW_EVIDENCE: YES,
+    ONE_OF: {
```

Comment:
Am I misunderstanding the logic, or is this `ONE_OF` block unnecessary since you already have `Q_2_H_2_NEW_EVIDENCE: YES`?